### ISSUE TITLE
cli push: creating strings that shouldn't be

### ISSUE THREAD

**Issue by jdewit** [transifex/transifex-javascript#107]
In a .ts file the following code is resulting in the creation of a new translation with string value "someInput".

```
import { Component, ElementRef, ViewChild } from '@angular/core';

@Component({
  selector: 'some-feature',
  templateUrl: './some-feature.html'
})
export class SomeFeature {
  @ViewChild('someInput', { static: true })
  someInputRef: ElementRef<any>;
}
```

**Comment by pablotransifex** [comment]
@jdewit We released a patch for this issue in 2.2.0, can you check if it's working for you now?

**Comment by jdewit** [comment]
Yeah, that fixed it. Thanks
